### PR TITLE
Launcher icon updated

### PR DESCRIPTION
**Pull request by Junaid55**
update the launcher icon plzz check

### CODE REVIEW

**Comment by Junaid55** [comment]
I an new in open source contribution so plzz help me and thanks for your
reply.

File: app/build.gradle [hunk 1]
```diff
@@ -1,4 +1,4 @@
-apply plugin: 'com.android.application'
+ apply plugin: 'com.android.application'
```
Comment (riturajjain2000) [inline app/build.gradle:1]: remove the leading space

File: app/build.gradle [hunk 2]
```diff
@@ -7,7 +7,7 @@ android {
 
     defaultConfig {
         applicationId "com.cswala.cswala"
-        minSdkVersion 26
+        minSdkVersion 23
```
Comment (riturajjain2000) [inline app/build.gradle:10]: already sone this , sync your branch with the latest develop branch

**Review by riturajjain2000 — CHANGES_REQUESTED** [review]
Make the PR to the develop branch!

**Review by riturajjain2000 — CHANGES_REQUESTED** [review]
It is still trying to be merged in main, plz Make PR to the develop branchh

**Pull request closed without merging**